lib/validation/rules/decimalOrHyphen.ts

```diff
@@ -1,8 +1,14 @@
-import { decimal as baseDecimal, helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
+import { decimal as baseDecimal, hyphen } from '../validators'
+
+export const message = {
+  en: 'The value must be valid decimal numbers or just a hyphen.',
+  ja: 'この値はハイフンまたは、小数および10進数である必要があります。'
```

Comment:
```suggestion
  ja: 'この値はハイフンまたは、小数または10進数である必要があります。'
```

Reply:
Fixed: e5c53aa

I made the order of words same as English error message.